File: Python/Baekjoon/10818.py
##1
import sys 
input =  sys.stdin.readline


N= int(input())
list_N=list(map(int, input().split()))

max = list_N[0]
min = list_N[0]

for i in list_N[1:]:
    if i > max:
        max = i
    elif i < min:
        min = i
    
print(min, max)
## 2
# n.sort()
# print(n[0],n[-1])

File: JavaScript/Baekjoon/10818.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = fs.readFileSync(filePath).toString().split("\n");

const n = Number(input[0]);
const list = input[1].split(" ");

let max = -1000000;
let min = 1000000;

for (let i = 0; i < n; i++) {
  let value = Number(list[i]);
  if (value < min) min = value;
  if (value > max) max = value;
}

console.log(`${min} ${max}`);
